main.qml

```diff
@@ -42,7 +42,7 @@ import "wizard"
 
 ApplicationWindow {
     id: appWindow
-
+    title: "Monero GUI"
```

Comment:
I used the title of the [README.md ](https://github.com/monero-project/monero-core/blob/master/README.md) file. Well, the title of the Monero wallet just says "Monero" when it's running. So we have basically three options: Monero, Monero Wallet or Monero GUI. Maybe @fluffypony or @Jaqueeee can help us with the decision?